services/web/server/tests/unit/with_dbs/02/test_projects_crud_handlers__list_with_query_params.py

```diff
@@ -360,3 +364,106 @@ async def test_list_projects_with_order_by_parameter(
     assert [
         item["description"][0] for item in data["data"]
     ] == _alphabetically_ordered_list
+
+
+@pytest.fixture()
+def setup_folders_db(
+    postgres_db: sa.engine.Engine,
+    logged_user: UserInfoDict,
+    user_project: ProjectDict,
+) -> Iterator[FolderID]:
+    with postgres_db.connect() as con:
+        output = []
+        result = con.execute(
+            folders.insert()
+            .values(
+                name="My Folder 1",
+                description="My Folder Decription",
+                product_name="osparc",
+                created_by=logged_user["primary_gid"],
+            )
+            .returning(folders.c.id)
+        )
+        _folder_id = result.fetchone()[0]
+
+        con.execute(
+            folders_to_projects.insert().values(
+                folder_id=_folder_id, project_uuid=user_project["uuid"]
+            )
+        )
+
+        yield FolderID(_folder_id)
+
+        con.execute(folders_to_projects.delete())
+        con.execute(folders.delete())
```

Comment:
here you delete everything, not just what you created. is there any reason why? (e.g. keeping the folder_ids created and then deleting where the c.folder_id in folder_ids?) Also is there no cascading between folders and folders_to_projects?